tests/test_license.py

```diff
@@ -85,7 +85,8 @@ def setUp(self):
             'src/opendr/perception/activity_recognition/x3d/algorithm',
             'src/opendr/perception/activity_recognition/cox3d/algorithm',
             'src/opendr/perception/object_tracking_2d/fair_mot/algorithm',
-            'src/opendr/perception/compressive_learning/multilinear_compressive_learning/algorithm/backbones'
+            'src/opendr/perception/compressive_learning/multilinear_compressive_learning/algorithm/backbones',
+            'src/opendr/perception/semantic_segmentation/bisenet/algorithm'
```

Comment:
Are there any changes to the original repo? If not, we should rather include the code as a submodule.
Any comments? @stefaniapedrazzi

Reply:
Here are all the cases:
* algorithm is copied from a third party: a `README.md` file has to be added in `src/opendr/perception/semantic_segmentation/bisenet/algorithm` mentioning the source and the license.
    * if code is available on a GitHub repo and no changes are applied, then it should be linked as submodule
    * otherwise if you applied any change or the code is not available on GitHub it is ok to include it directly in the OpenDR repo
* if copy is written from scratch and not copied, then the license test should not be skipped for the `algorithm` folder, but instead the OpenDR license has to be added to all the source files in the `algorithm` folder.